controllers/suite_test.go

```diff
@@ -415,3 +416,12 @@ func testObjectMeta(name string, namespace string, uid string) metav1.ObjectMeta
 		UID:       types.UID(uid),
 	}
 }
+
+func evaluateTestError(expected *string, actual error) {
+	if expected == nil {
+		Expect(actual).To(BeNil())
+	} else {
+		Expect(expected).ToNot(BeNil())
+		Ω(actual.Error()).Should(ContainSubstring(*expected))
```

Comment:
Could you change this to also use `Expect` for better consistency? I know it was just moved but would be nice to fix at the same time.
```suggestion
		Expect(actual.Error()).To(ContainSubstring(*expected))
```